metabase/metabase
# Convert audit v2 column_settings to column names

Depends on https://github.com/metabase/metabase/pull/46383

This is automatic replacement done by the migration script. Please note that audit v2 viz settings were designed to work with postgres and mysql only.

How to verify:
- Merge `column-settings-legacy` locally
- Run docker, then `yarn test-qa-dbs:up` to spin up postgres
- `MB_DB_TYPE=postgres MB_DB_PORT=5404 MB_DB_HOST=localhost MB_DB_DBNAME=sample MB_DB_USER=metabase MB_DB_PASS=metasample123 yarn dev-ee` to start Metabase with postgres
- Enter the token and active your Metabase EE
- Browse affected questions/models/dashboards

Note
- There were some broken settings that didn't point to any column in the response
- No need to check every changed entity
